src/python/pants/jvm/subsystems.py

```diff
@@ -79,3 +79,12 @@ class JvmSubsystem(Subsystem):
             """
         ),
     )
+    global_options = StrListOption(
+        "--global-options",
+        help=softwrap(
+            """
+            List of JVM options to pass to all JVM processes.
```

Comment:
It would probably be good to inline some of the details from the error message about how to set `-Xmx`.